partiql-spi/src/main/kotlin/org/partiql/spi/fn/builtins/FnOr.kt

```diff
@@ -25,9 +25,18 @@ internal object Fn_OR__BOOL_BOOL__BOOL : Fn {
     )
 
     override fun invoke(args: Array<Datum>): Datum {
-        if (args[0].isNull || args[1].isNull) return Datum.nullValue(PType.typeBool())
-        val lhs = args[0].boolean
-        val rhs = args[1].boolean
-        return Datum.bool(lhs || rhs)
+        val lhs = args[0]
+        val rhs = args[1]
+        val lhsIsUnknown = lhs.isNull || lhs.isMissing
+        val rhsIsUnknown = rhs.isNull || rhs.isMissing
```

Comment:
I think we should apply this change for `FnNot`?

Reply:
Added. Not exactly the same logic since the NULL CALL status handles it for `FnNot`, but I've added support for the missing check.